backends/tc/tc.def

```diff
@@ -379,9 +379,9 @@ class TCTable {
                 tcTable += "\n$TC p4template create table/" + pipelineName
                             + "/" + controlName + "/" + tableName
                             + " entry" + entry->toString();
+                tcTable += " permissions 0x1024";
```

Comment:
Are the permissions fixed as 0x1024 for all tables?

Reply:
yes